corehq/apps/domain/tests/test_views.py

```diff
@@ -241,44 +305,56 @@ def test_apply_command_with_other_doamin_alert(self):
         self.assertEqual(messages[0].message, 'Alert not found!')
         self.assertEqual(response.status_code, 302)
 
+
+class TestDeleteDomainAlertView(TestBaseDomainAlertView):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.url = reverse('delete_domain_alert', kwargs={
+            'domain': cls.domain_name,
+        })
+
+    def test_feature_flag_access_only(self):
+        response = self.client.post(self.url)
+        self.assertEqual(response.status_code, 404)
+
     @flag_enabled('CUSTOM_DOMAIN_BANNER_ALERTS')
-    def test_creating_new_alert(self):
-        self.assertEqual(Alert.objects.count(), 2)
+    def test_post_access_only(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 405)
 
+    @flag_enabled('CUSTOM_DOMAIN_BANNER_ALERTS')
+    def test_with_missing_alert_id(self):
+        with self.assertRaisesMessage(AssertionError, 'Missing alert ID'):
+            self.client.post(
+                self.url,
+                data={},
+            )
+
+    @flag_enabled('CUSTOM_DOMAIN_BANNER_ALERTS')
+    def test_with_missing_alert(self):
```

Comment:
simply added tests for missing alert and a successful delete after this but due to the diff, it looks like I updated an existing test.